e2e/jasmine_test/.bazelrc

```diff
@@ -0,0 +1,7 @@
+# Disable lockfile for now. It is unstable.
```

Comment:
nit: Fabian points out that the lockfile makes things faster for you locally, so we should maybe `.gitignore` them rather than disable them

Reply:
Hah. That was my original approach in rules_js and we scrapped that after review in favor of the disable 🤷

Reply:
Switched

Reply:
Yeah I was wrong, good to check with the real experts